remote.go

```diff
@@ -57,15 +59,15 @@ type remoteWD struct {
 // server.
 var HTTPClient = http.DefaultClient
 
-// jsonContentType is JSON content type.
-const jsonContentType = "application/json"
-
 func newRequest(method string, url string, data []byte) (*http.Request, error) {
 	request, err := http.NewRequest(method, url, bytes.NewBuffer(data))
 	if err != nil {
 		return nil, err
 	}
-	request.Header.Add("Accept", jsonContentType)
+	if data != nil {
+		request.Header.Add(headers.ContentType, mediatypes.ApplicationJSONUtf8)
```

Comment:
Since this was already broken, this needs a test. 

Maybe we can modify one of the existing tests to include UTF-8 data in the payload?